ISISComputingGroup/EPICS-DbUnitChecker
# Ticket2816

A bit of refactoring on the DB unit checker to check DBs more nicely and allow skipping only specific tests for specific DBs

https://github.com/ISISComputingGroup/IBEX/issues/2816

To test,

```
python C:\Instrument\Apps\EPICS\ISIS\DbUnitChecker\master\db_checker.py -i ioc support ISIS
```

And check that all tests pass

Also check that this fixes https://github.com/ISISComputingGroup/IBEX/issues/2821 and close if it does.
